### DISCUSSION TITLE
How to run the exported node graph in the script editor with code without opening nxt-editor in Maya?

### DISCUSSION THREAD

**Discussion by ordinc** [nxt-dev/nxt_editor#139] [Q&A] [answered]
As the title describes, I checked the related help documents (I don’t know if the document I checked is correct)
https://nxt-dev.github.io/api/nxt.html
I copied the sample code and executed it in Maya's script editor, and received an error.
![image](https://user-images.githubusercontent.com/35759494/105125890-70846680-5b18-11eb-8ca1-842878755ab7.png)

**Comment by ordinc** [comment]
I downloaded and updated Editor v3.6.0.
And the import rules of the import module are changed, as shown in the figure below. (The red underline is the change)
The node graph exported by nxt-editor was successfully run.
![image](https://user-images.githubusercontent.com/35759494/105139484-5905a780-5b31-11eb-8941-38499e04f457.png)

**Comment by ImLucasBrown** [comment, marked as answer]
While its not strictly _wrong_ to copy out the source and run it, it isn't the expected way to use the NXT package.
Can you confirm this issues exists when using the following 2 lines of code in the script editor:
```python
import nxt
nxt.execute_graph('graph/path.nxt')
```

**Reply by ordinc** [reply]
Thank you very much for your answers!
The two lines of code you provided did not have the above errors, and the node graph was successfully executed.
